src/Tooltip/tooltip.test.tsx

```diff
@@ -1,97 +1,111 @@
+import 'jest-enzyme'
+
 import React from 'react'
 import { shallow } from 'enzyme'
 import Tooltip, { formatCurrency, defaultDirection } from './index'
 
 describe('Tooltip', () => {
   describe('render()', () => {
     it('renders', () => {
-      expect(shallow(<Tooltip />).exists()).toBe(true)
+      expect(shallow(<Tooltip />)).toExist()
     })
 
     it('renders a transparent element when isActive is false', () => {
-      const tooltip = shallow(<Tooltip isActive={false} />)
-
-      expect(tooltip.exists()).toBeTruthy()
-      expect(tooltip.getElement().props.style.opacity).toEqual(0)
+      expect(
+        shallow(<Tooltip isActive={false} />)
+      ).toHaveStyle('opacity', 0)
     })
 
     it('renders an opaque element when isActive is true', () => {
-      const tooltip = shallow(<Tooltip isActive />)
-
-      expect(tooltip.exists()).toBeTruthy()
-      expect(tooltip.getElement().props.style.opacity).toEqual(1)
+      expect(
+        shallow(<Tooltip isActive />)
+      ).toHaveStyle('opacity', 1)
     })
 
     it('renders name', () => {
-      const tooltip = shallow<Tooltip>(<Tooltip name={'example-name'} />)
-
-      expect(tooltip.exists()).toBeTruthy()
-      expect(tooltip.text().includes(tooltip.instance().props.name)).toBeTruthy()
+      expect(
+        shallow(<Tooltip name={'example-name'} />)
+      ).toIncludeText(
+        'example-name'
```

Comment:
Is there a reason why we're breaking up all these lines like this?

Reply:
Just a personal preference. I think they're easier to read this way.